RELEASE/scripts/autoscend/autoscend_header.ash

```diff
@@ -532,16 +529,20 @@ boolean drinkSpeakeasyDrink(string drink);					//Defined in autoscend/auto_clan.
 boolean eatFancyDog(string dog);							//Defined in autoscend/auto_clan.ash
 boolean zataraClanmate(string who);							//Defined in autoscend/auto_clan.ash
 boolean zataraSeaside(string who);							//Defined in autoscend/auto_clan.ash
+boolean isActuallyEd();										//Defined in auto_ascend/auto_edTheUndying.ash
 float edMeatBonus();										//Defined in autoscend/auto_edTheUndying.ash
 boolean ed_buySkills();										//Defined in autoscend/auto_edTheUndying.ash
 boolean ed_autoAdv(int num, location loc, string option);		//Defined in autoscend/auto_edTheUndying.ash
 boolean ed_autoAdv(int num, location loc, string option, boolean skipFirstLife);//Defined in autoscend/auto_edTheUndying.ash
 boolean ed_doResting();										//Defined in autoscend/auto_edTheUndying.ash
 boolean ed_eatStuff();										//Defined in autoscend/auto_edTheUndying.ash
-boolean ed_handleAdventureServant(int num, location loc, string option);//Defined in autoscend/auto_edTheUndying.ash
+void ed_handleAdventureServant(location loc);//Defined in auto_ascend/auto_edTheUndying.ash
 void ed_initializeDay(int day);								//Defined in autoscend/auto_edTheUndying.ash
 void ed_initializeSession();								//Defined in autoscend/auto_edTheUndying.ash
 void ed_initializeSettings();								//Defined in autoscend/auto_edTheUndying.ash
+skill ed_nextUpgrade();										//Defined in auto_ascend/auto_edTheUndying.ash
```

Comment:
can get rid of this through `ed_needShop` if we scope them under `ed_shopping` though you'd need to check to make sure no one used `ed_needShop` somewhere by now that will break

Reply:
Done.